### ISSUE TITLE
Add possibility to use test permutation in RNG tests for mobile theme

### ISSUE THREAD

**Issue by oas004** [joreilly/Confetti#569]
Following up on https://github.com/joreilly/Confetti/pull/528#issuecomment-1500398908 . Currently the BaseScreenshotTest does not support to make multiple tests for each permutation of the mobile theme. This was supported in the old test classes. We should be able to support that somehow.

**Comment by yschimke** [comment]
I think so. But I'm not sure it's needs to be "library" code. As long as a test can be parameterised and control those aspects in its setup, we don't need an API

**Comment by oas004** [comment]
Yeah I was thinking to add it in each module so it  is not part of the test library module at least :)
